### PR TITLE
add testcases for MaintenanceMode

### PR DESCRIPTION
**Pull request by maryvictol**
This PR add gui testcases and for Maintenance mode https://github.com/epam/cloud-pipeline/issues/2423

### CODE REVIEW

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_4.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Check switch to Maintenance mode during the run Pausing and Resuming operations
+
+Test verifies that running Pausing and Resuming processes are proceeded at switch to the maintenance mode
+
+**Prerequisites**:
+- Admin user
+- Perform [2423_1](2423_1.md) case
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_4.md]: I think that this step is redundant. Else step 9 of the test case is unlogic.

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+| 8 | Set *Cloud Region* the same as *Region* specified for node pool at step 4 of the Preparations | |
+| 9 | Expand the **Advanced** section | | 
+| 10 | Select **Spot** Price type | |
+| 11 | Launch the run | |
+| 12 | At the **Runs** page, click the just-launched run | |
+| 13 | Expand the **Instance** section. Wait until the **IP** label appears | | 
+| 14 | Open the **Cluster state** page | |
+| 15 | Click the **HOT NODE POOLS** tab | **1/2** nodes is shown in the pool state for pool created at step 7 of the Preparations |
+| 16 | Click the node pool created at step 7 of the Preparations | One node has the label with the run ID launched at step 11 | 
+| 17 | Repeat steps 1-15 | **2/3** nodes is shown in the pool state for pool created at step 7 of the Preparations |
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md]: **2/3** nodes label doesn't appear right away. Possible, here the "waiting" step should be added?

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Check disabled Hot node pools autoscaling in Maintenance mode
+
+Test verifies that Hot node pools autoscaling is disabled in the maintenance mode.
+
+**Prerequisites**:
+- Admin user
+- Perform [2423_1](2423_1.md) case
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md]: I think that this step is redundant. Else step 22 of the test case is unlogic.

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+| 22 | Mark **Enabled** checkbox for **`system.maintenance.mode`** preference.  | |
+| 23 | Click **Save** button | |
+| 24 | Repeat steps 1-15 | Pool size is automatically increased: **3/3** nodes is shown in the pool state |
+| 25 | Repeat steps 1-15 | Pool size isn't automatically increased: **3/3** nodes is shown in the pool state |
+| 26 | Open the **Cluster state** page | The node for run launched at step 25 isn't contains the label with the pool name specified at step 7 of the Preparations |
+
+**After:**
+- Restore **`system.maintenance.mode`** preference to false value.
+- Restore **`system.maintenance.mode.banner`** preference to value saved at step 6 of the [2423_1](2423_1.md) case
+- Stop runs launched at step 11, 24 and 25
```
Comment (NShaforostov) [inline docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md]: Fix to "... steps 11, 17, 24 and 25"

**Review by NShaforostov — APPROVED** [review]
Fix minor notes before the merging

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md (hunk 4 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md]: Done

File: docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md (hunk 2 above)
Comment (maryvictol) [reply docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2423/2423_5.md]: Done

**Pull request merged**